recipes/ase/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ase" %}
+{% set version = "3.12.0" %}
+{% set sha256 = "e05cb18c4ce751750c9596ae302ef0828f1f259cc66f6b7e7e70d6d7c9f27bbd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy
+    - scipy
+  run:
+    - python
+    - numpy
+    - matplotlib
+    - scipy
+
+test:
+  imports:
+    - ase
+  commands:
+    - python -m ase.test
+  requires:
+    - qt
```

Comment:
It wasn't immediately obvious from grep'ing the code where `qt` was required to run the tests.